ci-operator/step-registry/hypershift/mce/heterogeneous/x86-power/hypershift-mce-heterogeneous-x86-power-workflow.yaml

```diff
@@ -0,0 +1,40 @@
+workflow:
+  as: hypershift-mce-heterogeneous-x86-power
+  documentation: |-
+    The hypershift-mce-heterogeneous-x86-power workflow executes tests against a new ephemeral Hypershift heterogeneous agent cluster
+    created on Power and x86 Nodes to validate the cluster creation reaches completed state.
+  steps:
+    pre:
+      - ref: ingress-aws-nlb-manifest
+      - chain: ipi-aws-pre
+      - ref: hypershift-mce-install
+      - ref: hypershift-mce-conf-os-images
+      - ref: hypershift-mce-power-create-hcp
+      - chain: hypershift-mce-heterogeneous-power-create-nodepool
+      - ref: hypershift-mce-heterogeneous-ibmcloud-x86-create-nodepool
+      - chain: hypershift-mce-heterogeneous-ibmcloud-x86-create-workers
```

Comment:
can we group this into a single chain?

Reply:
Grouping them into single chain will cause repetition.
IMHO, it is better we keep it as it is.